lib/compilation-queue.ts

```diff
@@ -42,8 +42,13 @@ const queueCompleted = new PromClient.Counter({
     help: 'Total number of jobs completed',
 });
 
+declare type Job<TaskResultType> = () => PromiseLike<TaskResultType>;
+
 export class CompilationQueue {
-    constructor(concurrency, timeout) {
+    private readonly _running: Set<number>;
```

Comment:
I think we could do the initialization here too

Reply:
Done; thanks!